docs/CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -218,6 +218,7 @@ For a detailed description of this feature, see the [Proposal: Custom CRD Downlo
 - add karmada.io/system=true label to internally created karmada cluster-roles and cluster-role-bindings. ([#5281](https://github.com/karmada-io/karmada/pull/5281), @grosser)
 - cluster-level resource propagation pause and resume capabilities. ([#4838](https://github.com/karmada-io/karmada/pull/4838), @a7i)
 - Adding FlinkDeployment v1beta1 CRD to supported third party resource customizatons. ([#5023](https://github.com/karmada-io/karmada/pull/5023), @mszacillo)
+- [Log mutating webhook handlers](https://github.com/karmada-io/karmada/pull/5521) for `OverridePolicy`, `ClusterPropagationPolicy`, and `MultiClusterService` webhooks.
```

Comment:
You need to put `Log mutating webhook handlers for `OverridePolicy`, `ClusterPropagationPolicy`, and `MultiClusterService` webhooks. `  into the PR body:  `Does this PR introduce a user-facing change? :`

and not update here.

Reply:
Not sure I understood correctly, you mean no need for release note for this PR?

Reply:
No, just edit here

![image](https://github.com/user-attachments/assets/a0a5b3e1-09f8-48ab-984b-75b5b5ecd8ad)

Reply:
Ah, got it! you mean the release notes would updated automatically if the release note in PR body once the PR get merged. Updated it now.

Reply:
> and then it can generate to CHANGELOG, it's generate by release manager.

I love the way it is automated!!